grind/bin/grind

```diff
@@ -398,6 +402,9 @@ class TestRunner:
         java_version = self.args.java_version or self.project_config.java_version
         assert java_version in supported_java_versions
 
+        if self.args.store_txt_output is True:
+         self.project_config.artifact_archive_globs = ["**/surefire-reports/TEST-*.xml", "**/surefire-reports/*.txt"]
```

Comment:
Indentation looks off here.

Also, it looks like this overrides the project_config globs? Shouldn't it append instead?

Reply:
Yes, you are right, I'll fix that. It should append. 
I don't know if there is a bug in the indentation, I'll check my settings in pycharm, using defaults.